apps/client/src/app/pages/home/header/Header.tsx

```diff
@@ -0,0 +1,47 @@
+import NotificationIcon from '@home-assets/notifications-icon.png';
+import UserIcon from '@home-assets/user-icon.png';
+import Button from '@map-shared-components/Button';
+
+function Header({ isLoggedIn, username }: HeaderProps) {
+  const handleClick = () => {};
+
+  return (
+    <div className='flex items-center justify-between bg-[#1D3C66] text-[#CADEED] py-4 px-6 font-roboto'>
+      <div className='flex items-center sm:space-x-4'>
+        <h1 className='text-xl font-bold'>Hermes</h1>
+        <div className='space-x-4 hidden sm:block'>
+          <Button onClick={handleClick}>Home</Button>
```

Comment:
These three buttons have the same behavior maybe you can create a common component for this, now these buttons don't have special functionalities but in the future maybe have, so is better to change 1 button than 3 of them